cdap-app-templates/cdap-etl/cdap-etl-tools/src/main/java/co/cask/cdap/etl/tool/UpgradeTool.java

```diff
@@ -174,8 +174,15 @@ private boolean upgradePluginTemplates(JsonObject consoleData) {
         for (Map.Entry<String, Map<String, Map<String, PluginTemplate>>> appEntry :
           namespaceEntry.getValue().entrySet()) {
           String appName = appEntry.getKey();
-          PluginArtifactFinder pluginArtifactFinder =
-            appName.equals(BATCH_NAME) ? batchPluginArtifactFinder : realtimePluginArtifactFinder;
+          PluginArtifactFinder pluginArtifactFinder;
+          if (BATCH_NAME.equals(appName)) {
+            pluginArtifactFinder = batchPluginArtifactFinder;
+          } else if (REALTIME_NAME.equals(appName)) {
+            pluginArtifactFinder = realtimePluginArtifactFinder;
+          } else {
+            // don't know what this is, ignore it.
```

Comment:
this isn't the pipeline upgrade section, but the draft upgrade section.  And if this value isn't cdap-etl-batch or cdap-etl-realtime, it won't show up in the 3.2.x anyway so having any messaging here would be more confusing than it is useful.